app/src/services/application/Application.js

```diff
@@ -0,0 +1,47 @@
+
+import _ from 'lodash';
+
+import committeesMap from 'common/committees';
+
+export class CommitteeApplication {
+  constructor(id=null, name, email, applicationText, committees, ordered){
+    this._name = name;
+    this._email = email;
+    this._applicationText = applicationText;
```

Comment:
I feel that the application object should match the format the API expects. Therefore, this should be `application_text` rather than `applicationText`. If the conversion is done only in `json()` it can become hard to spot errors with regards to transformation to a schema. 

If we're going to implement an admin interface we'll have to write a converter back from the json format anyways, and it's not like we're gonna be talking to multiple application backends at this point.